pages/styles/styles.css

```
@@ -0,0 +1,213 @@
+* {
+    font-family:Helvetica;
+}
+body {
+    background-color: pink;
+}
+/********** about grid container*************************** */
+.container-about {
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+}
+.cell {
+    color: white;
+    font-size: 1rem;
+    font-style: bold;
```

Bold is not a font style.  Instead it should be:

```suggestion
    font-weight: bold;
```